docs/docs/network/02-coordinator.md

```diff
@@ -27,35 +30,32 @@ ignite n publish https://github.com/ignite/example
 ⋆ Launch ID: 3
 ```
 
-`LaunchID` identifies the published blockchain on Ignite blockchain
+`LaunchID` identifies the published blockchain on Ignite blockchain.
 
-## List all published chains
+### Specify a initial genesis
 
-```
-ignite n chain list
-```
+During coordination, new genesis accounts and genesis validators are added into the chain genesis.
+The initial genesis where these accounts are added is by default the default genesis generated by the chain binary.
 
-### Output
+The coordinator can specify a custom initial genesis for the chain launch with the `--genesis` flag. This custom initial genesis can contain additional default genesis accounts and custom params for the chain modules.
 
-```
-Launch Id 	Chain Id 	Source ...
+A URL must be provided for the `--genesis` flag. This can either directly point to a JSON genesis file or a tarball containing a genesis file.
 
-3 		    example-1 	https://github.com/ignite/example
-2 		    spn-10 		https://github.com/tendermint/spn
-1 	        example-20 	https://github.com/tendermint/spn
+```shell
+ignite n publish https://github.com/ignite/example --genesis https://example.com/genesis.tar.gz
 ```
 
----
-
 ## Approve validator requests
 
-First, list requests:
+When coordinating for a chain launch, validators send requests. These represent requests to be part of the genesis as a validator the chain.
```

Comment:
Missing the word "of". `When coordinating for a chain launch, validators send requests. These represent requests to be part of the genesis as a validator of the chain.`

Reply:
Latestl version is `When coordinating for a chain launch, validators send requests. These represent requests to be part of the genesis as a validator for the chain.`